packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -1436,8 +1436,11 @@ export class Install {
             try {
                 // find the adapter's io-package.json
                 const adapterNpm = `${tools.appName.toLowerCase()}.${adapter}`;
+                const ioPackPath = require.resolve(`${adapterNpm}/io-package.json`, {
+                    paths: tools.getDefaultRequireResolvePaths(module)
+                });
                 // eslint-disable-next-line @typescript-eslint/no-var-requires
-                const ioPack = require(`${adapterNpm}/io-package.json`); // yep, it's that easy
+                const ioPack = require(ioPackPath);
```

Comment:
Should we maybe change to `fs-extra.readJsonSync`? I guess we don’t need require cache here and it would be more straightforward for reading json